docs/troubleshooting.rst

```diff
@@ -0,0 +1,45 @@
+Troubleshooting
+===============
+
+Aqueduct in Kubernetes
+==============================================
+
+If you encounter memory issues while using the Aqueduct tool in Kubernetes, such as OOM killer terminating processes within a pod, follow this troubleshooting guide.
```

Comment:
We need to describe that mainly causes problem - it's oom , its not depend if user use aqueduct or not.